napari/_qt/widgets/qt_dims_slider.py

```diff
@@ -413,7 +413,7 @@ def _play(
             _connect={'frame_requested': self.qt_dims._set_frame},
         )
         worker.finished.connect(self.qt_dims.stop)
-        thread.finished.connect(self.play_stopped.emit)
+        thread.finished.connect(self.play_stopped)
```

Comment:
@tlambert03 : can you provide any insight on why this change might have fixed those interesting test failures.